pkgs/by-name/ex/extract-vmlinux/package.nix

```diff
@@ -0,0 +1,58 @@
+# Extracts <linux-latest-src>/scripts/extract-vmlinux as package. As this script
+# is rather basic and generic, we do not need to care about the specific Linux
+# src version from that we get the script.
+
+{ lib
+, linux_latest # kernel package to get the source tree from
```

Comment:
Usually for this sort of thing we use linuxHeaders (see e.g. bpftools), because we don't really want things depending on linux_latest.